module/Activity/view/activity/activity-calendar/index.phtml

```diff
@@ -12,12 +12,13 @@ $form->prepare();
 <div class="container option-calendar">
     <div class="row">
         <h1><?= $this->translate('Option calendar') ?></h1>
-        <?= $this->translate('Welcome on the page of the digital Option Calendar. In the calendar below you can place options for GEWIS activities, just like this was done before on the paper version of the option calendar. The red dots indicate activities that are already in the GEWIS agenda, the blue dots are options for activities. If you insert an option, please give it a clear name, such that it is clear where the option is for.') ?>
+        <?= $this->translate('Welcome on the page of the digital Option Calendar. In the calendar below you can place options for GEWIS activities, just like you were used to do in the paper version of the option calendar before. The red dots indicate activities that are already fixed in the GEWIS agenda, the blue dots are options for activities. If you insert an option, please give it a clear name, so it is clear where the option is for.') ?>
```

Comment:
"so it is clear where the option is for", seems that where should be what. Nice Dunglish though.